src/data/officers.json

```diff
@@ -90,159 +90,160 @@
     },
 
     {
       "name": "Anikait Sharma",
       "image": "/static/images/former-officers/anikait_sharma.jpg",
-      "major": "Computer Science",
+      "major": "Computer Science '21",
       "position": "Vice President",
       "term": "2020 - 2021",
-      "description": "Anikait has a keen interest in HCI on how we can use technology to increase accessibility for all users. If Anikait is not at Zach procrastinating on his code - he’s probably climbing some rocks, cooking some grub, or concert hopping with his homies!"
+      "description": "Anikait has a keen interest in HCI on how we can use technology to increase accessibility for all users. If Anikait was not at Zach procrastinating on his code - he’s probably climbing some rocks, cooking some grub, or concert hopping with his homies!"
     },
 
     {
       "name": "Hannah Gooden",
       "image": "/static/images/former-officers/hannah_gooden.jpg",
-      "major": "Computer Science",
+      "major": "Computer Science '20",
       "position": "President",
       "term": "2018 - 2020",
-      "description": "Hannah has two minors in astrophysics and cybersecurity, and is very interested in the way scientists use computing to do research. She also loves traveling, and spent a whole summer in New Mexico, where she crewed for a hot air balloon team on her weekends."
+      "description": "Hannah earned two minors in astrophysics and cybersecurity. She loves traveling, and spent a whole summer in New Mexico, where she crewed for a hot air balloon team on her weekends. She is now at the Johns Hopkins University APL!"
     },
 
     {
       "name": "Marissa Soria",
       "image": "/static/images/former-officers/marissa_soria.jpg",
-      "major": "Electronic Systems Engineering Technology",
+      "major": "Electronic Systems Engineering Technology '21",
       "position": "Events Chair",
       "term": "2018 - 2020",
-      "description": "Marissa has a special interest in cyber security and hardware integration and in her free time enjoys baking and gardening."
+      "description": "Marissa has a special interest in cyber security and hardware integration and in her free time enjoys baking and gardening. She is now a Senior Consultant at Booz Allen Hamilton!"
     },
 
     {
       "name": "Feras Khemakhem",
       "image": "/static/images/former-officers/feras_khemakhem.jpg",
-      "major": "Computer Science",
+      "major": "Computer Science '20",
       "position": "Project Management Chair",
       "term": "2017 - 2020",
-      "description": "Feras specializes in Computer Graphics and animation, and hopes to continue this work into the field. Personal hobbies include traveling to eat good food, and gaming."
+      "description": "Feras completed his Masters in Computer Science in 2022 at A&M working on Computer Graphics and Animation the field. Personal hobbies include traveling to eat good food, and gaming. He is now a Software Engineer at Microsoft!"
```

Comment:
I think we can remove "the field' and also change 'Personal hobbies' to  'His personal hobbies'